src/PYTHON/pair_python.cpp

```diff
@@ -392,6 +392,17 @@ void PairPython::coeff(int narg, char **arg)
 
 /* ---------------------------------------------------------------------- */
 
+void PairPython::init_style()
+{
+  if (did_dummy_restart)
+    error->all(FLERR,"Must specify 'pair_style' command after "
+               "'read_restart' for pair style python");
+
+  neighbor->request(this,instance_me);
```

Comment:
@akohlmey looks like there's a missing neighbor.h here

Reply:
should be addressed in dcf2a33